src/kbmod/search/RawImage.cpp

```diff
@@ -27,6 +27,11 @@ RawImage::RawImage() {
     pixels = std::vector<float>();
 }
 
+RawImage::RawImage(const RawImage& old) {
```

Comment:
I added a move ctor and both copy and move assignments.

There is destructor already in the header file. It is just at the bottom (not sure why, but I never moved it).

The explicit tag causes problems with the copy constructor and pybind11. I'm not sure why.

Reply:
I believe it can cause some problems with templates. Apologies. It was for the other uses but I don't think it is hugely important. I noticed it is recommended by lsst.developer.io